src/components/Buttontest.css

```diff
@@ -17,4 +18,6 @@
   cursor: pointer;
   font-size: 16px;
   text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.5);
+  font-size: 1.5vw;
```

Comment:
Better to have a constant font size, instead of a constantly scaling one. Make the width of the button scale constantly, not the font-size itself. (Use a media query)